Wrappers/Python/cil/optimisation/functions/TotalVariation.py

```diff
@@ -132,35 +199,33 @@ def regularisation_parameter(self, value):
 
     def __call__(self, x):
         
-        r''' Returns the value of the \alpha * TV(x)'''
+        r""" Returns the value of the TotalVariation function at :code:`x` ."""
+
         try:
             self._domain = x.geometry
         except:
             self._domain = x
-        # evaluate objective function of TV gradient
+
+        # Compute Lipschitz constant provided that domain is not None.
+        # Lipschitz constant dependes on the GradientOperator, which is configured only if domain is not None
+        if self._L is None:
+            self.calculate_Lipschitz()
+
+        if self.strong_convexity_constant>0:
+            tmp = (self.strong_convexity_constant/2)*x.squared_norm()
```

Comment:
could you use a more descriptive name than `tmp` or  add a comment on what `tmp` is.

Reply:
Renamed to `strongly_convex_term`
https://github.com/TomographicImaging/CIL/blob/fcbe5f25d3e6e952a4240a8ed0a492213d678324/Wrappers/Python/cil/optimisation/functions/TotalVariation.py#L213